src/main/java/appeng/integration/modules/crafttweaker/InscriberRecipes.java

```diff
@@ -0,0 +1,108 @@
+
+package appeng.integration.modules.crafttweaker;
+
+
+import java.util.Collections;
+
+import net.minecraft.item.ItemStack;
+
+import crafttweaker.IAction;
+import crafttweaker.api.item.IItemStack;
+import stanhebben.zenscript.annotations.Optional;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+import appeng.api.AEApi;
+import appeng.api.features.IInscriberRecipe;
+import appeng.api.features.IInscriberRecipeBuilder;
+import appeng.api.features.InscriberProcessType;
+
+
+@ZenClass( "mods.appliedenergistics2.Inscriber" )
+public class InscriberRecipes
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1118)